app/src/commander.cpp

```diff
@@ -57,7 +57,37 @@ void CmdHandler::handle(CmdSetUsbChannelMsg& msg)
         }
     }
 }
-
+//------------------------------------------------------------------------------
+void CmdHandler::handle(CmdGetPwrRelayMsg& msg)
+{
+    if (!m_pwrRelays) // Check if m_pwrRelays has been created
+    {
+        err("PowerRelays object has not been created, exiting...");
+    }
+    else
+    {
+        // Check if ID is valid ?
+        if (msg.relayId >= m_pwrRelays->size())
+        {
+            err("PwrRelayId value must be between (0-%d) !!!\r\n", m_pwrRelays->size()-1);
+        }  
+        else
+        {
+            // Extract a PwrRelay reference.
+            auto& pwrRelay = (*m_pwrRelays)[msg.relayId];
+            const PowerRelay::RelayState m_state = pwrRelay.state();

```

Comment:
maybe refactor it to a single inf(...) without an if statement?

Reply:
done